packages/ui-module/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
Can we change this to be a .eslintrc.js file? That way we keep the standard used in the root.

Reply:
Well, no problem😃